lib/taskana-simplehistory-example/src/main/java/pro/taskana/simplehistory/TaskanaHistoryPluginConfiguration.java

```diff
@@ -10,8 +10,9 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Primary;
 
-import pro.taskana.history.plugin.configuration.TaskanaHistoryEngineConfiguration;
-import pro.taskana.history.plugin.impl.TaskanaHistoryEngineImpl;
+import pro.taskana.configuration.TaskanaEngineConfiguration;
+import pro.taskana.history.api.TaskanaHistory;
+import pro.taskana.simplehistory.impl.HistoryEventServiceImpl;
 
 public class TaskanaHistoryPluginConfiguration {
```

Comment:
We don't call it "Plugin" anymore, do we? Maybe you can change this to the new naming scheme? E.g. "TaskanaSimpleHistoryProviderConfiguration"?

Reply:
Agree